database/rrdhost.c

```diff
@@ -812,6 +813,136 @@ struct label *load_config_labels()
     return l;
 }
 
+static inline void strip_last_symbol(char *str, char symbol)
+{
+    char *end = str;
+
+    while (*end && *end != symbol)
+        end++;
+    if (*end && *end == symbol)
```

Comment:
I think you do not need to compare again the `while` condition here, it is enough to check whether `*end == symbol`, because case it is not, this already means that `*end` is `NULL`.

Reply:
Removed an unnecessary check.